documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+mongodb           active      1  mongodb  dpe/edge   96  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        <host IP>    27017/tcp  Replica set primary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+
+```
+Use the IP address listed underneath `Public address` for `mongodb/0` as your host.
+
+**Retrieving the database name:** In this case we are connecting to the `admin` database. Use `admin` as the database name. Once we access the database via the MongoDB URI, we will create a `test-db` database to store data.
+
+**Retrieving the replica set name:** The replica set name is the name of the application on Juju hosting MongoDB. The application name in this tutorial is `mongodb`. Use `mongodb` as the replica set name. 
+
+### Connect via MongoDB URI
+Now that we have the necessary fields to connect to the URI, we can connect to MongoDB via the URI. Enter the following into the command line, replace the values for  `username`, `password`, `hosts`, `database name`, and the `replica set name` with what you’ve retrieved above:
+```shell
+mongosh mongodb://<username>:<password>@<host>/<database name>?replicaSet=<replica set name>
+```
+
+You should now see:
+```
+Current Mongosh Log ID:	6389e2adec352d5447551ae0
+Connecting to:		mongodb://<credentials>@10.23.62.156/admin?replicaSet=mongodb&appName=mongosh+1.6.1
+Using MongoDB:		5.0.14
+Using Mongosh:		1.6.1
+
+For mongosh info see: https://docs.mongodb.com/mongodb-shell/
+
+
+To help improve our products, anonymous usage data is collected and sent to MongoDB periodically (https://www.mongodb.com/legal/privacy-policy).
+You can opt-out by running the disableTelemetry() command.
+
+------
+   The server generated these startup warnings when booting
+   2022-12-02T11:24:05.416+00:00: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine. See http://dochub.mongodb.org/core/prodnotes-filesystem
+------
+
+------
+   Enable MongoDB's free cloud-based monitoring service, which will then receive and display
+   metrics about your deployment (disk utilization, CPU, operation statistics, etc).
+
+   The monitoring data will be available on a MongoDB website with a unique URL accessible to you
+   and anyone you share the URL with. MongoDB may use this information to make product
+   improvements and to suggest MongoDB products and deployment options to you.
+
+   To enable free monitoring, run the following command: db.enableFreeMonitoring()
+   To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
+------
+
+mongodb [primary] admin>
+```
+
+You can now interact with MongoDB directly using any [MongoDB commands](https://www.mongodb.com/docs/manual/reference/command/). For example entering `show dbs` should output something like:
```

Comment:
Will you be covering any of those commands later in the tutorial? Or is this left as an "exercise to the reader"?

Reply:
Some of these commands are shown in the tutorial, but not all of them. (This is a MongoDB specific thing so its less related to the charm itself)

For the sake of learning we walk users though some basic commands, but we assume there is already a familiarity with MongoDB